lib/galaxy/managers/users.py

```diff
@@ -271,14 +271,14 @@ def by_api_key(self, api_key, sa_session=None):
             raise exceptions.AuthenticationFailed("Provided API key has expired.")
         return provided_key.user
 
-    def check_master_api_key(self, api_key):
-        master_api_key = getattr(self.app.config, "master_api_key", None)
-        if not master_api_key:
+    def check_bootstrap_admin_api_key(self, api_key):
+        bootstrap_admin_api_key = getattr(self.app.config, "bootstrap_admin_api_key", None)
+        if not bootstrap_admin_api_key:
             return False
         # Hash keys to make them the same size, so we can do safe comparison.
-        master_hash = hashlib.sha256(util.smart_str(master_api_key)).hexdigest()
+        bootstrap_hash = hashlib.sha256(util.smart_str(bootstrap_admin_api_key)).hexdigest()
```

Comment:
```suggestion
        bootstrap_hash = hashlib.sha256(util.smart_str(self.app.config.bootstrap_admin_api_key)).hexdigest()
```
(if you accept the previous suggestion.)